src/main/java/publishers/EmailPublisher.java

```diff
@@ -32,6 +32,12 @@
 
 public class EmailPublisher {
 
+	private Properties props2;
+	private Session session2;
+	private String username2;
+	private Properties props1;
+	private Session session1;
+	private String username1;
 	private Properties props;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "props" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1450)